setup.py

```diff
@@ -68,7 +68,7 @@ class SpecialDependencies:
         "idaes-pse==2.0.0a3",
     ]
     for_prerelease = [
-        "idaes-pse @ https://github.com/gmlc-dispatches/idaes-pse/archive/2.0.0.dev3.dispatches.22.10.06.zip",
+        "idaes-pse @ https://github.com/dguittet/idaes-pse/archive/update_grid_integration.zip"
```

Comment:
```suggestion
        "idaes-pse @ https://github.com/gmlc-dispatches/idaes-pse/archive/2.0.0.dev4.dispatches.22.10.26.zip"
```

@dguittet I've just created the `2.0.0.dev4.dispatches.22.10.26` tag corresponding to `IDAES/idaes-pse:main` up to and including (the merge commit for) IDAES/idaes-pse#986.

Reply:
@lbianchi-lbl there is yet another fix we needed in IDAES https://github.com/IDAES/idaes-pse/pull/996 to get all the double loop notebooks to work. could you please merge that one and make another tag? This PR is now being re-tested with the idaes branch above, but I will switch to the dispatches/idaes tag

Reply:
@dguittet I've just created `2.0.0.dev4.dispatches.22.10.26.1` that includes IDAES/idaes-pse#996.

Reply:
Thanks! updated